src/services/compare-mmdl/handlers/getMmdlData.ts

```diff
@@ -40,11 +45,11 @@ exports.handler = async function (
 };
 
 // 'DD/MM/YYYY'
-function getSecsSinceToday(date: string) {
-  const today = new Date().getTime();
+function getSecsSinceNow(date: string | number) {
```

Comment:
string should be removed here, right?

Reply:
i use it for both string and number on 25 and 29

Reply:
I didn't realize `new Date` could handle a string.  Thanks.